pywbemtools/pywbemcli/_cmd_class.py

```diff
@@ -175,10 +175,10 @@ class hierarchy defined by `CLASSNAME` argument if provided.
 
     The output format is defined by the output-format general option.
 
-    The includeclassqualifiers, includeclassorigin options define optional
+    The local-only and no-qualifiers options define optional
```

Comment:
Show option names local-only and no-qualifiers with leading dashes and in example font: `--local-only` and `--no-qualifiers`.